OData/src/System.Web.Http.OData/OData/Routing/ODataRoute.cs

```diff
@@ -105,12 +105,25 @@ public override IHttpVirtualPathData GetVirtualPath(HttpRequestMessage request,
         /// Headers for the next max version will be ignored.
         /// </summary>
         /// <returns>Returns itself so that multiple calls can be chained.</returns>
+        [Obsolete("Suggest to use API SetODataVersionConstraint")]
         public ODataRoute HasRelaxedODataVersionConstraint()
         {
+            return SetODataVersionConstraint(true);
+        }
+
+        /// <summary>
+        /// Set flag to enable/disable the version constraint relaxing.
+        /// When enable relaxing. The service will allow clients to send both OData V1 to V3 and next max version headers.
+        /// Headers for the next max version will be ignored.
+        /// </summary>
+        /// <param name="isRelaxedMatch">True for enable, false for disable</param>
+        /// <returns>Returns itself so that multiple calls can be chained.</returns>
+        public ODataRoute SetODataVersionConstraint(bool isRelaxedMatch)
```

Comment:
`EnableRelaxedVersionConstraint`?  Also add this to a public API doc or GitHub page for 5.7.